doc/api/corepack.md

```diff
@@ -12,10 +12,8 @@ added:
 
 > Stability: 1 - Experimental
 
-Documentation for this tool can be found on the [Corepack repository][].
+**Corepack will no longer be distributed with Node.js 25+.**
+Users currently depending on the `corepack` from Node.js should switch
```

Comment:
```suggestion
Users currently depending on the bundled `corepack` module from Node.js should switch
```

Reply:
Done in [`ef47418` (#57825)](https://github.com/nodejs/node/pull/57825/commits/ef47418b51edca56560e43b698079660d06208ef)